tests/unit/utils/toHoursAndMinutes.spec.js

```diff
@@ -1,19 +1,15 @@
 import toHoursAndMinutes from "@/utils/toHoursAndMinutes";
 
-const input = 71;
-const output = toHoursAndMinutes(input);
-
 describe("toHoursAndMinutes", () => {
-  it("returns value", () => {
-    expect(output).toBeTruthy;
+  it("returns 'Unknown' when number is less than or equal to 0", () => {
+    expect(toHoursAndMinutes(0)).toMatch("Unknown");
   });
 
-  it("returns string", () => {
-    expect(typeof output).toBe("string");
+  it("returns time in minutes when number is less than 60", () => {
+    expect(toHoursAndMinutes(59)).toMatch("59min");
   });
 
   it("return string in correct format", () => {
-    const re = /\dh\s\d{2}min/;
-    expect(output).toMatch(re);
+    expect(toHoursAndMinutes(71)).toMatch("1h 11min");
```

Comment:
I suggest to add one more expect to this test just to make sure:
```js
expect(toHoursAndMinutes(60)).toMatch( ? );
expect(toHoursAndMinutes(71)).toMatch("1h 11min");
```

Reply:
thank you!!! 👀

Reply:
precisely speaking, it was a suggestion to check how your function works exactly for `60`, because (after reading the tests) I have the impression that this is not what is expected 😉

Reply:
@judytabaca see above comment (that's the last piece to finish it) :)

Reply:
Artur my biggest apologies, i've went through the task again yesterday - fingers crossed this is looking better now :)

Reply:
Nothing to apologize for, great job I just wanted it to be merged already! 🚀